packages/@aws-cdk/aws-lambda-destinations/lib/event-bridge.ts

```diff
@@ -17,7 +17,7 @@ export class EventBridgeDestination implements lambda.IDestination {
   /**
    * Returns a destination configuration
    */
-  public bind(_scope: Construct, fn: lambda.IFunction): lambda.DestinationConfig {
+  public bind(_scope: Construct, fn: lambda.IFunction, _type?: lambda.DestinationType): lambda.DestinationConfig {
```

Comment:
Let's wrap this in an "options" struct (after 2 positional arguments, we move to keyword arguments)